extensions/pyRevitTools.extension/checks/cad_audit_check.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+
+from pyrevit import script, revit, DB, DOCS
+from pyrevit.forms import alert
+from pyrevit.preflight import PreflightTestCase
+
+from System.Windows import Window # Used for cancel button
+from rpw.ui.forms import (FlexForm, Label, Separator, Button) # RPW
+from rpw.ui.forms.flexform import RpwControlMixin, Controls # Used to create RadioButton
+
+from pyrevit.coreutils import Timer # Used for timing the check
+from datetime import timedelta # Used for timing the check
+
+doc = DOCS.doc
+ac_view = doc.ActiveView
+
+
+def collect_cadinstances(active_view_only):
+    """     Collect ImportInstance class from whole model or from just active view  """
+    if active_view_only:
+        cadinstances = DB.FilteredElementCollector(doc, ac_view.Id).OfClass(DB.ImportInstance).WhereElementIsNotElementType().ToElements()
+    else:
+        cadinstances = DB.FilteredElementCollector(doc).OfClass(DB.ImportInstance).WhereElementIsNotElementType().ToElements()
```

Comment:
reduce code repetition
```suggestion
    collector = DB.FilteredElementCollector(doc, ac_view.Id) if active_view_only else DB.FilteredElementCollector(doc)
    cadinstances = collector.OfClass(DB.ImportInstance).WhereElementIsNotElementType().ToElements()
```